### DISCUSSION TITLE
競賽資訊分享- 2023 CareerHack台積電校園黑客松

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS3073-C/Python-Community-GS3073-C#527] [General]
![https://careercenter.ncu.edu.tw/storage/files/2022-12-22T09:35:00+08:00/IT%20CareerHack%20Poster%20(8).png](https://careercenter.ncu.edu.tw/storage/files/2022-12-22T09:35:00+08:00/IT%20CareerHack%20Poster%20(8).png)

這次的台積校園黑客松分為四個主題可以報名，包括**智慧停車**、**Chatbot**、**網路服務架構設計**、**AI影像智慧**，其中聊天機器人與AI影像智慧，是跟課程內容比較相關的，另外，比賽說明也有強調到**雲端平台**、**Machine Learning**、**Data Analysis**、**DevOps**的重要，而雲端平台和Machine Learning也是課程中有提及的，所以對大家應該算比較好入門的比賽，大家可以上競賽官網去參考看看~ 

https://www.tsmc.com/static/english/careers/2023Careerhack/index.html

**Comment by NcuJerryKao** [comment]
好強大的競賽 這種我從沒想過參加
不過對於像助教這麼強的工程師 應該是先看活動獎項 前幾名能得到什麼XDDD

**Comment by williamyen0623** [comment]
我還只是個小菜雞，黑客松太困難了，不過是個很好的目標！

**Comment by BingRong1217** [comment]
感覺好讚好想參加！

**Comment by Ben6uo** [comment]
這類競賽感覺就是給高難關卡阿
